spec/redis_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+def configure_test_redis_db
+  url = ENV.fetch("REDIS_URL", "redis://localhost:6379/6") # AnyCable uses Redis DB number 5 by default
```

Comment:
`redis://localhost:6379/*6*` and ` AnyCable uses Redis DB number 5 by default`? :thinking:
```suggestion
  url = ENV.fetch("REDIS_URL", "redis://localhost:6379/5") # AnyCable uses Redis DB number 5 by default
```